docs/src/tutorials/bifurcation_diagram.md

```diff
@@ -1,77 +1,58 @@
 # Bifurcation Diagrams
-Bifurcation diagrams can be produced from Catalyst generated models through the use of the [BifurcationKit.jl](https://github.com/rveltz/BifurcationKit.jl/) package. This tutorial gives a simple example of how to create such a bifurcation diagram. *Please note,* this tutorial is written for BifurcationKit version 0.1.11 and has not yet been updated to work on newer versions like 0.2 and up.
+Bifurcation diagrams can be produced from Catalyst generated models through the use of the [BifurcationKit.jl](https://github.com/rveltz/BifurcationKit.jl/) package. This tutorial gives a simple example of how to create such a bifurcation diagram. 
 
-First, we declare our model. For our example we will use a bistable switch, but which also contains a Hopf bifurcation.
-```julia
+First, we declare our model. For this example we will use a bistable switch, but one which also contains a Hopf bifurcation.
+```@example ex1
 using Catalyst
 rn = @reaction_network begin
     (v0+v*(S*X)^n/((S*X)^n+(D*A)^n+K^n),d), ∅ ↔ X
-    (τ*X,τ), ∅ ↔ A
+    (X/τ,1/τ), ∅ ↔ A
 end S D τ v0 v K n d
 ```
-Next, BifurcationKit requires another form for the system function and Jacobian than what is used by the SciML ecosystem, so we need to declare these:
-```julia
-odefun = ODEFunction(convert(ODESystem,rn),jac=true)
-F = (u,p) -> odefun(u,p,0)      
-J = (u,p) -> odefun.jac(u,p,0)
+Next, we specify the system parameters for which we wish to plot the bifurcation diagram. We also set the parameter we wish to vary in our bifurcation diagram, as well as over which interval. Finally, we set which variable we wish to plot the steady state values of in the diagram.
+```@example ex1
+p = [:S=>1., :D=>9., :τ=>1000., :v0=>0.01, :v=>2., :K=>20., :n=>3, :d=>0.05]
```

Comment:
If you make this a `Dict` then won't your `p_bstart` calculation become trivial? ModelingToolkit should support `Dict`s for the mappings too.